src/sass/rivet.scss

```diff
@@ -13,6 +13,7 @@
 @use 'breadcrumb';
 @use 'buttons';
 @use 'buttons-segmented';
+@use 'calendar';
```

Comment:
We might think about calling this something like `calendar-tile`. My only concern is that it might be confused with an actual functional calendar/datepicker component.